src/components/carousel/_index.scss

```
@@ -0,0 +1,159 @@
+@use "../../scss";
+@use "~slick-carousel/slick/slick";
+
+.c-carousel {
+	@include scss.component-properties("carousel");
+
+	/* Arrows */
+	.slick-prev,
+	.slick-next {
+		cursor: pointer;
+		font-size: 0;
+		line-height: 0;
+		position: absolute;
+		top: 50%;
+		display: block;
+
+		&:hover {
+			color: transparent;
+			background: transparent;
+
+			&:before {
+				@include scss.component-properties("carousel-buttons-hover-before");
+			}
+		}
+
+		&:focus {
+			color: transparent;
+			background: transparent;
+
+			&:before {
+				@include scss.component-properties("carousel-buttons-focus-before");
+			}
+		}
+
+		@include scss.component-with-states("carousel-buttons", [hover, focus]);
+	}
+
+	.slick-next {
+		&:before {
+			font-family: "Arial", serif;
+			content: ">";
+			-webkit-font-smoothing: antialiased;
+			-moz-osx-font-smoothing: grayscale;
+		}
+
+		@include scss.component-with-states("carousel-button-next", [before]);
+	}
+
+	.slick-prev {
+		&:before {
+			font-family: "Arial", serif;
+			content: "<";
+			-webkit-font-smoothing: antialiased;
+			-moz-osx-font-smoothing: grayscale;
+		}
+
+		@include scss.component-with-states("carousel-button-previous", [before]);
+	}
+
+	.slick-prev.slick-disabled {
+		&:before {
+			opacity: 0.25;
+		}
+
+		@include scss.component-with-states("carousel-button-previous-disabled", [before]);
+	}
+
+	.slick-next.slick-disabled {
+		&:before {
+			opacity: 0.25;
+		}
+
+		@include scss.component-with-states("carousel-button-next-disabled", [before]);
+	}
+
+	[dir="rtl"] {
+		.slick-prev {
+			left: auto;
+
+			&:before {
+				content: ">";
+			}
+
+			@include scss.component-with-states("carousel-button-rtl-previous", [before]);
+		}
+
+		.slick-next {
+			right: auto;
+			@include scss.component-properties("carousel-button-next");
+
+			&:before {
+				content: "<";
+			}
+
+			@include scss.component-with-states("carousel-button-rtl-next", [before]);
+		}
+	}
+
+	/* Dots */
+	.slick-dotted.slick-slider {
+		@include scss.component-properties("carousel-slider-margin");
```

I think this is the one that was 'bottom' before?  Can we just drop the `-margin` part and call it the `carousel-slider`?  Let the theme determine if it's going to add margin, border, animations or whatnot.  I don't think we should "lead on" the use of the component part for a particular style.
```suggestion
		@include scss.component-properties("carousel-slider");
```

---

See updated code base